flink-quickstart/flink-quickstart-java/src/main/resources/archetype-resources/pom.xml

```diff
@@ -50,181 +51,113 @@ under the License.
 		</repository>
 	</repositories>
 
-	<!-- 
-
-		Execute "mvn clean package -Pbuild-jar"
-		to build a jar file out of this project!
-
-		How to use the Flink Quickstart pom:
-
-		a) Adding new dependencies:
-			You can add dependencies to the list below.
-
-		b) Build a jar for running on the cluster:
-
-			"mvn clean package -Pbuild-jar"
-			This will create a fat-jar which contains all dependencies necessary for running the created jar in a cluster.
-	-->
-
 	<dependencies>
 		<!-- Apache Flink dependencies -->
-		<dependency>
-			<groupId>org.apache.flink</groupId>
-			<artifactId>flink-core</artifactId>
-			<version>${flink.version}</version>
-		</dependency>
+		<!-- These dependencies are provided, because they should not be packaged into the JAR file. -->
 		<dependency>
 			<groupId>org.apache.flink</groupId>
 			<artifactId>flink-java</artifactId>
 			<version>${flink.version}</version>
+			<scope>provided</scope>
 		</dependency>
 		<dependency>
-			<!-- This dependency is required to actually execute jobs. It is currently pulled in by
-				flink-streaming-java, but we explicitly depend on it to safeguard against future changes. -->
 			<groupId>org.apache.flink</groupId>
-			<artifactId>flink-clients_${scala.binary.version}</artifactId>
```

Comment:
is this just for simplification or do we no longer expect that flink-streaming-java looses it's dependency on flink-clients?